devices/tuya.js

```diff
@@ -454,6 +453,24 @@ module.exports = [
             {vendor: 'Mercator Ikuü', model: 'SSWD01'},
         ],
     },
+    {
+        fingerprint: [
+            {modelID: 'TS0601', manufacturerName: '_TZE200_3p5ydos3'},
+        ],
+        model: 'TS0601_dimmer',
```

Comment:
I expect that the other `TS0601_dimmer` will also support this feature (if not we can always split them later)

Reply:
I didn't dare changing the code for ALL Tuya dimmers, especially, since the min brightness of 3p5ydos3 is using the data point other dimmers are using as the brightness level...

Well... I moved the exposes definition of min- and max-brightness to the existing model TS0601_dimmer, but I think the better way would be to create a new model like TS0601_dimmer_with_min_max or alike...

Reply:
Okay... I reverted the description, fixed the tests of tuya_data_point_dump in test/fromZigbee.test.js and corrected a bunch of Lint errors. All checks passed now.